### PR TITLE
Updating content to match prototype 9

### PR DESCRIPTION
**Pull request by cbroadley-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/confirm-identity-verification/confirm-identity-verification.njk [hunk 1]
```diff
@@ -10,6 +10,15 @@
 
 {% include "partials/user_name.njk" %}
 <h1 class="govuk-heading-l">{{ i18n.confirmVerificationHeading }}</h1>
+
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+      <span class="govuk-visually-hidden">Warning</span>
```
Comment (ttingle-ch) [inline src/views/confirm-identity-verification/confirm-identity-verification.njk:17]: This warning will need translating as it is read aloud by screen readers

File: src/views/index/home.njk [hunk 2]
```diff
@@ -6,11 +6,21 @@
...
-      {{ i18n.guidePageVerificationServiceNote }}
+    <p class="govuk-body">{{ i18n.guidePageVerificationServiceNote }}
       <a href="https://www.gov.uk/guidance/companies-house-identity-verification-standard" id="identity-verification-standard-link">{{ i18n.linkGuidePageVerificationService }}</a>.
+    </p>
+    <p class="govuk-body">{{ i18n.keepRecordsOfTheIdentityChecks }}</p>
+
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-visually-hidden">Warning</span>
```
Comment (ttingle-ch) [inline src/views/index/home.njk:19]: As above

File: src/views/confirm-identity-verification/confirm-identity-verification.njk (hunk 1 above)
Comment (cbroadley-ch) [reply src/views/confirm-identity-verification/confirm-identity-verification.njk]: Added text to common.json locales file to account for Welsh translation

File: src/views/index/home.njk (hunk 2 above)
Comment (cbroadley-ch) [reply src/views/index/home.njk]: Added text to common.json locales file to account for Welsh translation

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Review by swalke-ch — APPROVED** [review]

**Pull request merged**